pkg/config/containers.conf

```diff
@@ -237,6 +237,15 @@ default_sysctls = [
 #
 #prepare_volume_on_create = false
 
+# Give extended privileges to this container. A privileged container
+# turns off the security features that isolate the container from the
+# host. Dropped Capabilities, limited devices, read-only mount points,
+# Apparmor/SELinux separation, and Seccomp filters are all disabled.
+#
+# Rootless containers cannot have more privileges than the user that
+# launched them.
+#privileged=true
```

Comment:
Should have an extra empty comment line like the other options.
Also this options should have the default set so privileged=false.